stats/stats/src/read.rs

```diff
@@ -0,0 +1,212 @@
+use chrono::{NaiveDate, NaiveDateTime};
+use sea_orm::{DatabaseConnection, DbBackend, DbErr, FromQueryResult, Statement};
+use stats_proto::blockscout::stats::v1::{Counters, LineChart, Point};
+use std::collections::HashMap;
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ReadError {
+    #[error("database error {0}")]
+    DB(#[from] DbErr),
+    #[error("chart {0} not found")]
+    NotFound(String),
+}
+
+#[derive(FromQueryResult)]
+struct CounterData {
+    name: String,
+    date: NaiveDateTime,
```

Comment:
Why here is `NaiveDateTime` used while in the `chart_data_int` entity it is a `Date` type?